app/models/public_body.rb

```diff
@@ -438,7 +438,10 @@ def self.import_csv_from_file(csv_filename, tag, tag_behaviour, dry_run, editor,
         # Give an error listing ones that are to be deleted
         deleted_ones = set_of_existing - set_of_importing
         if deleted_ones.size > 0
-          notes.push "Notes: Some " + tag + " bodies are in database, but not in CSV file:\n    " + Array(deleted_ones).sort.join("\n    ") + "\nYou may want to delete them manually.\n"
+          notes.push "Notes: Some #{tag} bodies are in database, but " \
+                     "not in CSV file:\n    " +
+                     Array(deleted_ones).compact.sort.join("\n    ") +
+                     "\nYou may want to delete them manually.\n"
```

Comment:
This might have been a missing migration - the public_body_translations table doesn't have the default values from public_bodies so translated fields could be returning nil instead of ""